docs/provider/hashicorp-vault.md

```diff
@@ -43,6 +43,24 @@ Then create a simple k/v pair at path `secret/foo`:
 vault kv put secret/foo my-value=s3cr3t
 ```
 
+If kv engine is not enabled then enable using:
+
+```
+vault secrets enable -path=secret kv-v2
+```
+
+If the KV version is version:1, upgrade it to version:2:
+
+```
+vault kv enable-versioning secret/
+```
+
+Can check version using following and check for `Options` column, it should indicate [version:2]:
+
+```
+vault secrets list -detailed
+```
+
```

Comment:
```suggestion
Can check version using following and check for `Options` column, it should indicate [version:2]:

```
vault secrets list -detailed
```

If you are using version: 1, jsut remember to update your SecretStore manifest appropriately
```

Reply:
It seems I cannot properly edit this one, but I think the idea is here :)

Reply:
Added comment on SecretStore manifest itself. As there's typo 'jsut' on above suggestion, replaced with 'just'